<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgLayerProperties</class>
 <widget class="QWidget" name="WdgLayerProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>190</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>&amp;Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>editName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel3">
       <property name="text">
        <string>&amp;Opacity:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>intOpacity</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KisSliderSpinBox" name="intOpacity" native="true"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="textLabel4">
       <property name="text">
        <string>Composite mode:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblColorSpaces">
       <property name="text">
        <string>Color space:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="lblColorSpace">
       <property name="text">
        <string comment="KDE::DoNotExtract">TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblProfiles">
       <property name="text">
        <string>Profile:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="lblProfile">
       <property name="text">
        <string comment="KDE::DoNotExtract">TextLabel</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KisCompositeOpComboBox" name="cmbComposite"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>126</width>
       <height>78</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBoxPreview">
     <property name="text">
      <string>&amp;Preview</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="grpActiveChannels">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Select the set of active channels.</string>
     </property>
     <property name="whatsThis">
      <string>Select the set of active channels. Only active channels will be visible, filtered or affected by painting.</string>
     </property>
     <property name="title">
      <string>&amp;Active Channels</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisSliderSpinBox</class>
   <extends>QWidget</extends>
   <header>kis_slider_spin_box.h</header>
  </customwidget>
  <customwidget>
   <class>KisCompositeOpComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/kis_cmb_composite.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
